app/models/course/group_user.rb

```diff
@@ -16,6 +16,12 @@ class Course::GroupUser < ApplicationRecord
   belongs_to :course_user, inverse_of: :group_users
   belongs_to :group, class_name: Course::Group.name, inverse_of: :group_users
 
+  scope :order_by_course_user_name, lambda {
+                                      joins('LEFT OUTER JOIN course_users ON '\
+                                            'course_users.id = course_group_users.course_user_id').
+                                        order('name ASC')
```

Comment:
Hmm do we also need to specify that the `name` column is to be read from the `course_users` table?

Reply:
Not really.. In fact, there's an error if we use course_users.name:
```
ActiveRecord::StatementInvalid:
       PG::DuplicateAlias: ERROR:  table name "course_users" specified more than once
```

Reply:
Ooo nice, thanks for the clarification!